sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1566,9 +1595,9 @@ def _retry_zones(
                     self._ensure_cluster_ray_started(handle, log_abs_path)
 
                 config_dict['handle'] = handle
-                plural = '' if num_nodes == 1 else 's'
-                logger.info(f'{fore.GREEN}Successfully provisioned or found'
-                            f' existing VM{plural}.{style.RESET_ALL}')
+                logger.info(
+                    ux_utils.finishing_message(
+                        f'Cluster {cluster_name!r} launched.', log_path))
```

Comment:
Hmm, when is this shown? Seems to always show `Cluster launched: xxx`

Reply:
Ahh, this is for legacy clouds using the old code path (node_provider/ray autoscaler) to provisioned. Changed the message to be consistent.